rust/datafusion/src/datasource/datasource.rs

```diff
@@ -24,6 +24,15 @@ use crate::arrow::datatypes::SchemaRef;
 use crate::error::Result;
 use crate::physical_plan::ExecutionPlan;
 
+/// The table statistics
+#[derive(Clone)]
+pub struct Statistics {
+    /// The number of table rows
+    pub num_rows: i64,
+    /// total byte of the table rows
+    pub total_byte_size: i64,
```

Comment:
```suggestion
    /// total size of the table, in bytes
    pub total_byte_size: u64,
```

Reply:
i64 was my suggestion to be consistent with https://github.com/apache/arrow/blob/master/rust/parquet/src/file/metadata.rs#L193

Perhaps that should be change to be `u64` too.

Reply:
> i64 was my suggestion to be consistent with https://github.com/apache/arrow/blob/master/rust/parquet/src/file/metadata.rs#L193
> 
> Perhaps that should be change to be `u64` too.

maybe, we should be change it from https://github.com/sunchao/parquet-format-rs/blob/f1d56730e24c06c9cdc39bfdb8000131469f28db/src/parquet_format.rs#L3258